packages/okta-angular/index.ts

```diff
@@ -1,13 +0,0 @@
-/*!
- * Copyright (c) 2017-Present, Okta, Inc. and/or its affiliates. All rights reserved.
- * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
- *
- * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *
- * See the License for the specific language governing permissions and limitations under the License.
- */
-
-export * from './src/';
```

Comment:
Sanity check that removing this file doesn't affect existing ways of importing the module? E.g. this won't break anyone upgrading from an earlier version of the package?

Reply:
This will not break anything, as nobody would/should require a .ts file directly, but rather use "import" from our module. The main entry point is defined in package.json (there are several entry points depending upon ES level, "fesm5" variant seems to be preferred by current versions of angular). The use of 'barrel' files seems implcated in the AOT failure. I don't think this file was part of the problem but avoiding any`index.ts` files seems to be a good convention for typescript projects. The main entry point is now `okta-angular.ts`